rjc_account_form/data/paper_format_tax_invoice.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
```

Comment:
this record is used?
value same as `paperformat_rjc_invoice`. you should be used 1 file.